cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines"]
+= Running workloads and buildah as user namespaces on {pipelines-title}
+include::_attributes/common-attributes.adoc[]
+:context: running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines
+
+toc::[]
+
+Running {pipelines-shortname} as root on a container can expose the host and container processes to other resources. In some cases, to reduce this type of exposure, you can run some workloads in a user namespace instead. In other cases, you may still need to run the workload as root on the container but still be secure on the host.
```

Comment:
```suggestion
Running {pipelines-shortname} as root on a container can expose the host and container processes to other resources. In some cases, to reduce this type of exposure, you can run some workloads in a user namespace instead. In other cases, you might still need to run the workload as root on the container but still be secure on the host.
```
Per ISG: `May: Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.`